tests/components/transmission/test_service.py

```diff
@@ -0,0 +1,30 @@
+"""Transmission service platform tests."""
+from tests.common import patch
+
+
+async def call_service(
```

Comment:
Please don't create a helper for calling services. Just do it inline in the tests.